src/API/Reports/Products/DataStore.php

```diff
@@ -402,6 +402,8 @@ public static function sync_order_products( $order_id ) {
 
 		$order_items = $order->get_items();
 		$num_updated = 0;
+		$decimals    = wc_get_price_decimals();
+		$round_tax   = 'no' === get_option( 'woocommerce_tax_round_at_subtotal' );
```

Comment:
Nice!  I'm wondering if it's worth keeping a list somewhere of all the settings that might affect our data and require re-syncing.

Not something that has to be handled here, but long-term it might be nice to know what requires a re-sync.

Reply:
Good idea. Created #3108 .